OJP/OJP_Common.xsd

```diff
@@ -283,11 +283,12 @@
 			</xs:element>
 		</xs:sequence>
 	</xs:group>
-	<xs:complexType name="SharingServiceStructure">
+	<xs:complexType name="AlternativeServiceStructure">
```

Comment:
See below the `SharingModel` element and structure - do we still need them, given "stationLess" etc. in `AlternativeModesOfOperationEnumeration`?

Reply:
Currently it is a bit more detailed. So I would leave it in the system. If the group would like to have it removed, I can certainly do that. @skinkie  @Aurige  @herlitze

Reply:
I am open to keeping and/or removing. No show stopper.

Reply:
For me it is also fine to leave the structure.

Reply:
will go through it.